jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -55,9 +55,16 @@
 public final class StackWalker {
 
 	private static final int DEFAULT_BUFFER_SIZE = 1;
-	private final static int J9_RETAIN_CLASS_REFERENCE = 1;
-	private final static int J9_SHOW_REFLECT_FRAMES = 2;
-	private final static int J9_SHOW_HIDDEN_FRAMES = 4;
+
+	/* Java StackWalker flags map to internal flags defined in java_lang_StackWalker.cpp */
```

Comment:
I don't think "map to" conveys the message that these constants must have the same values as the similarly named macros in `java_lang_StackWalker.cpp`. I also think it would be better if the names weren't just "similar", but rather identical. Please update the comment (here and `java_lang_StackWalker.cpp` and remove the `J9_` prefixes here.

Reply:
The `J9_` prefix is used to differentiate with the StackWalker's Option enum.
I've updated both Java and native code to use `J9_` prefix and updated the comment to reflect that they are equivalent to each other

Reply:
Adding the `J9_` prefix is a better plan.